starport/pkg/localfs/search_test.go

```diff
@@ -0,0 +1,115 @@
+package localfs
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func setupGlobTests(t *testing.T, files []string) string {
+	t.Helper()
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	tmpdir, err := os.MkdirTemp(dir, "glob-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		_ = os.RemoveAll(tmpdir)
+	})
+	for _, file := range files {
+		fileDir := filepath.Dir(file)
+		fileDir = filepath.Join(tmpdir, fileDir)
+		if _, err := os.Stat(fileDir); os.IsNotExist(err) {
```

Comment:
If error is not nil and error is not related to the existence of the dir, test should still needs to fail.